File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_10.py
altura = int(input('Digite sua altura:(ex 170)'))
peso = int(input('Digite seu peso:(ex:65)'))

imc = (peso / (altura**2))*10000

if imc < 18.5:
    print(f'IMC: {imc:.2f} | Situação: Abaixo do peso!')
elif imc > 18.5 and imc < 25:
    print(f'IMC: {imc:.2f} | Situação: Peso normal.')
elif imc > 25 and imc < 30:
    print(f'IMC: {imc:.2f} | Situação: Sobrepeso!.')
elif imc > 30 and imc < 35:
    print(f'IMC: {imc:.2f} | Obesidade Grau 1.')
elif imc > 35 and imc < 40:
    print(f'IMC: {imc:.2f} | Obesidade Grau 2.')
else:
    print(f'IMC: {imc:.2f} | Obesidade Grau 3.')

File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-10.js
const prompt = require('prompt-sync')()

const peso = prompt('Peso:')
const altura = prompt('Altura:')

const IMC = (10000*(peso / (altura**2))).toFixed(2)

console.log(`\nIMC:${IMC}`)

if(IMC <= 18.5){
    return console.log('Abaixo do Peso')
}else if(IMC > 18.5 && IMC <25){
    return console.log('Peso Normal')
}else if(IMC >= 25 && IMC <30){
    return console.log('Sobrepeso')
}else if(IMC >= 30 && IMC <35){
    return console.log('Obesidade grau 1')
}else if(IMC >= 35 && IMC <40){
    return console.log('Obesidade grau 2')
}else if(IMC >= 40){
    return console.log('Obesidade grau 3')
}else{
    return console.log('Tem algo muito errado')
}